huxley/core/migrations/0025_auto_20171105_1635.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-11-05 16:35
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [('core', '0024_auto_20170927_2010'), ]
```

Comment:
Move this dependency forward past the newer migrations (0026 is the most recent).

Reply:
Oops, forgot to commit new dependencies

Reply:
Also what do you mean "move forward"? I don't know a lot about how migrations and dependencies work.

Reply:
By move forward I just meant make it dependent on the most recent migration. Its not really proper terminology.

Reply:
Ok sounds good! Also just for my own knowledge, what's the point of doing this?

Reply:
Looking into it more, guessing it has to do with wanting this migration to appear after the migrations for new features already merged onto the master branch (like the position paper grading you've been working on)